go/vt/vtctl/reparentutil/util.go

```diff
@@ -343,3 +343,20 @@ func waitForCatchUp(
 	}
 	return nil
 }
+
+// GetBackupCandidates is used to get a list of healthy tablets for backup
+func GetBackupCandidates(tablets []*topo.TabletInfo, stats []*replicationdatapb.Status) (res []*topo.TabletInfo) {
+	for i, stat := range stats {
+		// Always include TabletType_PRIMARY
+		if tablets[i].Type == topodatapb.TabletType_PRIMARY {
+			res = append(res, tablets[i])
+			continue
+		}
+		// shardTablets[i] and stats[i] is 1:1 mapping
+		// Healthy shardTablets[i] will be added to tablets
+		if stat != nil {
+			res = append(res, tablets[i])
+		}
```

Comment:
suggest condensing this into either a `switch` or `if/else` and avoiding the `continue`

Reply:
alternatively:

```go
if tablets[i].Type == topodatapb.TabletType_PRIMARY || stats != nil {
    res = append(...)
}
```

Reply:
Good approach.